sass/main.scss

```diff
@@ -344,6 +348,53 @@ nav {
 
     }
 
+    #modal {
+        display: none;
+        position: fixed;
+        z-index: 100;
+        padding: 100px;
+        left: 0;
+        top: 0;
+        width: 100%;
+        height: 100%;
+        overflow: auto;
+        background-color: rgba(black, 0.9);
+    }
+
+    #modal-content {
+        margin: auto;
+        display: block;
+        width: 80%;
+        max-width: 700px;
+        filter: blur(0);
```

Comment:
Does this line do anything?

Reply:
I thought it would make the image clearer during transition clear, lol. I don't know if it worked; my eyes are not the best.

Reply:
Let's remove it!